Dragonflight/WarriorArms.lua

```diff
@@ -852,29 +913,34 @@ spec:RegisterAbilities( {
     execute = {
         id = 163201,
```

Comment:
Does Massacre no longer cause Execute to change to a different spell (281000)?

Reply:
Yup your right, will fix.
PTR:
arms massacre: 281000
arms non: 163201
prot massacre: 281000
prot non: 163201
fury massacre: 280735
fury non: 5308